packages/@webex/plugin-meetings/src/roap/request.ts

```diff
@@ -96,7 +97,12 @@ export default class RoapRequest extends StatelessWebexPlugin {
       `Roap:request#sendRoap --> ${locusSelfUrl} \n ${roapMessage.messageType} \n seq:${roapMessage.seq}`
     );
 
-    Metrics.postEvent({event: eventType.MEDIA_REQUEST, meetingId});
+    NewMetrics.submitClientEvent({
```

Comment:
same like my other comment - we should move this into LocusMediaRequest.sendHttpRequest()

Reply:
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-440552